index.bs

```diff
@@ -65,16 +65,19 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
+The device orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
 
-Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
-of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
+The device orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
+of the [=low-level=] motion sensors (accelerometer, gyroscope). The type of fusion (complementary
```

Comment:
"Game Orientation Sensor" is basically gyroscope data presented as a quaternion. It's the one I was referring to here: https://github.com/w3c/sensors/issues/170#issuecomment-282325398, but couldn't recall the Android name for. Its unique goals is ultra-low latency. As a tradeoff it drifts and doesn't know where the north is. :D

"Geographic orientation", on the other hand, needs the magnetometer to find the north. Magnetometers are slow and messy, so the controller has much higher latency. It can rely on either a magnetometer or a gyroscope for the rest of the data (or even a combination of both). Android uses a magnetometer for this as their consumption is lower than gyroscopes and the use case for such sensors are more about indoor location (i.e. things that potentially can happen in the background and stay on for long, chewing away at the battery). There is no GPS involved.

FWIW, my opinion is still the same on this topic: we should be taking a holistic approach to the motion sensors and strive for consistency and clarity. The platform's been waiting for these for quite a while, would be a shame to rush it and repeat the same mistakes.